pkg/values/k3s.go

```diff
@@ -10,10 +10,11 @@ import (
 )
 
 var K3SVersionMap = map[string]string{
-	"1.28": "rancher/k3s:v1.28.2-k3s1",
-	"1.27": "rancher/k3s:v1.27.6-k3s1",
-	"1.26": "rancher/k3s:v1.26.9-k3s1",
-	"1.25": "rancher/k3s:v1.25.14-k3s1",
+	"1.29": "rancher/k3s:v1.29.0-k3s1",
+	"1.28": "rancher/k3s:v1.28.5-k3s1",
+	"1.27": "rancher/k3s:v1.27.9-k3s1",
+	"1.26": "rancher/k3s:v1.26.12-k3s1",
+	"1.25": "rancher/k3s:v1.25.16-k3s1",
```

Comment:
Can we remove the 1.25 version since it's active and maintenance support ended?
https://endoflife.date/kubernetes